### PR TITLE
Changes for proxy URLs

### PR DESCRIPTION
**Pull request by yannisle**

### CODE REVIEW

File: devices/hololens/hololens-offline.md [hunk 1]
```diff
@@ -35,9 +33,126 @@ HoloLens need a network connection to go through initial device set up.  If your
...
+|                                                     | *.dl.delivery.mp.microsoft.com                                      |   |   |   |
+|                                                     | *.windowsupdate.com                                                 |   |   |   |
+|                                                     | *.delivery.mp.microsoft.com                                         |   |   |   |
+|                                                     | *.update.microsoft.com                                              |   |   |   |
+
+
+
+## References
+
+>[!NOTE]
```
Comment (illfated) [inline devices/hololens/hololens-offline.md]: May I suggest using consistent spacing for MarkDown quote indentations:
```suggestion
> [!NOTE]
```

**Comment by illfated** [comment]
Please notice that the single space between the reference link text [in square brackets] and its (URL link) breaks your intended text linking.

File: devices/hololens/hololens-offline.md [hunk 2]
```diff
@@ -17,13 +17,11 @@ appliesto:
 - HoloLens 2
 ---
```
Comment (scooley) [inline devices/hololens/hololens-offline.md]: I can't edit but this breaks a ton of formatting standards.  Can you update to adhere to the standards in https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=master 

One heading (# heading) at the top of the page.  Please make all of the sub headings L2 again (## second level heading) and remove spaces from table.

**Comment by illfated** [comment]
@yannisle : Would you like individual suggestions for each line that needs changing?
Or are you able to make the edits yourself, based on the previous comments above?

File: devices/hololens/hololens-offline.md [hunk 3]
```diff
@@ -35,9 +33,125 @@ HoloLens need a network connection to go through initial device set up.  If your
 | MSA | https://login.live.com/ppsecure/inlineconnect.srf?id=80600 |
 | MSA Pin | https://account.live.com/msangc?fl=enroll |
 
-Additional references:
+## Use HoloLens online
```
Comment (scooley) [inline devices/hololens/hololens-offline.md]: Service specific endpoints?  Use online isn't accurate.

File: devices/hololens/hololens-offline.md [hunk 4]
```diff
@@ -35,9 +33,125 @@ HoloLens need a network connection to go through initial device set up.  If your
 | MSA | https://login.live.com/ppsecure/inlineconnect.srf?id=80600 |
 | MSA Pin | https://account.live.com/msangc?fl=enroll |
 
-Additional references:
+## Use HoloLens online 
+
+To take full advantage of HoloLens functionality, the following endpoints need to be enabled in your network configuration.
```
Comment (scooley) [inline devices/hololens/hololens-offline.md]: Update to be clearer this is a reference for enabling different, cloud tied, hololens experiences without full access to the internet?

My hesitation is that this list certainly isn't exhaustive.

**Comment by scooley** [comment]
@illfated - I think this is ready to merge.  Anything else needed to merge?

**Comment by yannisle** [comment]
@illfated do we need anything else for this to go live?

**Comment by illfated** [comment]
Thank you for asking. It looks good to me now (LGTM).
Rich diff, for better overview of the changes:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5927/files?short_path=5a46716#diff-5a46716896de56b5de229dbe728f3dfd

**Comment by scooley** [comment]
@DaniHalfin - this is ready to merge :)

**Pull request merged**